.github/workflows/upload.yml

```diff
@@ -2,21 +2,22 @@ name: upload
 
 on:
   push:
-    branches:
-      - master
+    tags:
+      - 'v*.*.*'
 
 jobs:
   upload:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: install dependencies
-        run: python3 -m pip install --user --upgrade setuptools wheel
+        run: python3 -m pip install --user --upgrade poetry
       - name: build
-        run: python3 setup.py sdist bdist_wheel
-      - name: install jfrog
-        run: curl -fL https://getcli.jfrog.io | sh
-      - name: upload to jfrog
-        run: | 
-          ./jfrog rt config --url "${{ secrets.ARTIFACTORY_URL }}" --user "${{ secrets.ARTIFACTORY_USER }}" --access-token "${{ secrets.ARTIFACTORY_PASSWORD }}" --interactive=false
-          ./jfrog rt u --build-name=Crypto-Blockchain-PYPI --build-number=${{ github.run_id }} --flat=true "dist/*" "${{ secrets.ARTIFACTORY_NAME }}"
\ No newline at end of file
+        run: poetry build
+      - name: release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            dist/*
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
before the security team use asked us to use jfrog to upload to pypi, peotry can do the same, right?

Reply:
For this part, I copied this GitHub workflow from `pystarport`. I checked with Huang Yi earlier, he said we just need to run the upload workflow and release on GitHub, then security team will pick up from there.